tests/System.IO.Compression.Tests/BrotliPerformanceCanterburyCorpus.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+using Microsoft.Xunit.Performance;
+using System.Collections.Generic;
+using System.IO.Compression;
+using System.Runtime.CompilerServices;
+using System.Security.Cryptography;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.IO.Compression.Tests
+{
+    public class BrotliPerformanceCanterburyCorpus
+    {
+        public static IEnumerable<object[]> CanterburyCorpus()
+        {
+            foreach (CompressionLevel compressionLevel in Enum.GetValues(typeof(CompressionLevel)))
+            {
+                foreach (int innerIterations in new int[] { 1, 10 })
+                {
+                    yield return new object[] { innerIterations, "alice29.txt", compressionLevel };
+                    yield return new object[] { innerIterations, "asyoulik.txt", compressionLevel };
+                    yield return new object[] { innerIterations, "cp.html", compressionLevel };
+                    yield return new object[] { innerIterations, "fields.c", compressionLevel };
+                    yield return new object[] { innerIterations, "grammar.lsp", compressionLevel };
+                    yield return new object[] { innerIterations, "kennedy.xls", compressionLevel };
+                    yield return new object[] { innerIterations, "lcet10.txt", compressionLevel };
+                    yield return new object[] { innerIterations, "plrabn12.txt", compressionLevel };
+                    yield return new object[] { innerIterations, "ptt5", compressionLevel };
+                    yield return new object[] { innerIterations, "sum", compressionLevel };
+                    yield return new object[] { innerIterations, "xargs.1", compressionLevel };
+                }
+            }
+        }
+
+        /// <summary>
+        /// Benchmark tests to measure the performance of individually compressing each file in the
+        /// Canterbury Corpus
+        /// </summary>
+        [Benchmark]
+        [MemberData(nameof(CanterburyCorpus))]
+        public void Compress_Canterbury(int innerIterations, string fileName, CompressionLevel compressLevel)
```

Comment:
> Should I change it?

I would say yes since I think that code was written prior to us having the inner iteration count feature and hence this naming conflict wasn't there. However, I don't have a strong opinion either way.